src/SarifWeb/Views/ValidationUi/Index.cshtml

```diff
@@ -57,15 +57,19 @@
                 <div id="ruleDescription"></div>
             </div>
             <div id="rightToolbarGroup">
-                <button id="downloadLogButton" aria-label="Download log file" title="Download log file" class="toolbar-button" disabled>
-                    <span class="ui-icon ui-icon-download">icon</span>
-                </button>
                 <button id="revalidateButton" aria-label="Re-validate log file" title="Re-validate log file" class="toolbar-button" disabled>
                     <span class="ui-icon ui-icon-arrowrefresh-1-e">icon</span>
                 </button>
                 <button id="utilitiesButton" aria-label="Utilities menu" title="Utilities menu" class="toolbar-button" disabled>
                     <span class="ui-icon ui-icon-gear">icon</span>
                 </button>
+                <div id="ruleFilters">
+                    <input type="checkbox" id="noteRules" aria-label="Rules with level 'note'" title="Rules with level 'note'" disabled />
```

Comment:
I think`aria-labelledby` might be more appropriate?

Reply:
Yes, you're [right](https://www.symphonious.net/2010/12/07/aria-labelledby-vs-aria-label/). Fixed.

---
In reply to: [467328176](https://github.com/microsoft/sarif-website/pull/115#discussion_r467328176) [](ancestors = 467328176)